runtime/frequency/src/lib.rs

```diff
@@ -17,16 +17,23 @@ pub fn wasm_binary_unwrap() -> &'static [u8] {
 	)
 }
 
+use common_runtime::constants::currency::UNITS;
+#[allow(unused)] // compiler lies
```

Comment:
Suggestion only: The use is under this feature flag if you want to limit the import on other places.
```suggestion
#[cfg(any(not(feature = "frequency-no-relay"), feature = "frequency-lint-check"))]
```